internal/app/bot/bot_client.go

```diff
@@ -167,3 +176,31 @@ func (c *BotClient) sendMessage(messages interface{}) {
 		}
 	}
 }
+
+func updatePhrases(update tgbotapi.Update) {
```

Comment:
I'd like to offer you to extract this function to another package, for example you can name it `mkrschi` and put it into `internal/app`, like `fakenews` or `string_builder`.
Just because I believe that this function is not in its place.